pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -101,9 +104,8 @@ func New(imageFsInfoProvider ImageFsInfoProvider, rootPath string, cgroupRoots [
 		Interval:     &duration,
 		AllowDynamic: pointer.Bool(allowDynamicHousekeeping),
 	}
-
 	// Create the cAdvisor container manager.
-	m, err := manager.New(memory.New(statsCacheDuration, nil), sysFs, housekeepingConfig, includedMetrics, http.DefaultClient, cgroupRoots, nil /* containerEnvMetadataWhiteList */, "" /* perfEventsFile */, time.Duration(0) /*resctrlInterval*/)
+	m, err := manager.New(memory.New(statsCacheDuration, nil), sysFs, housekeepingConfig, includedMetrics, http.DefaultClient, cgroupRoots, strings.Split(*envMetadataWhiteList, ",") /* containerEnvMetadataWhiteList */, "" /* perfEventsFile */, time.Duration(0) /*resctrlInterval*/)
```

Comment:
Just wondering if `containerEnvMetadataWhiteList` is a better name for this variable? It looks like this name was actually used here, but commented out.

Reply:
envMetadataWhiteList here is copied from https://github.com/google/cadvisor/blob/688835d77b199a4e95497b7d7d7b3580d5b03860/cmd/cadvisor.go#L68.